plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -198,6 +198,26 @@ export function AboutContent(props: AboutContentProps) {
           <Chip key={t} size="small" label={t} />
         ))}
       </AboutField>
+      <AboutField
+        label="Labels"
+        value="No Labels"
+        gridSizes={{ xs: 12, sm: 6, lg: 4 }}
+      >
+        {(Object.keys(entity?.metadata?.labels || {}) as (keyof string)[]).map(
+          t => (
+            <Chip
+              key={t as string}
+              size="small"
+              label={`${t as string}:${
```

Comment:
Perhaps a space after the `:` for a more clear divider?

Reply:
Hi,
Thanks @Rugvip for the suggestion. I have made the necessary changes in the code and updated it.